### DISCUSSION TITLE
Has this community moved back to Discourse?

### DISCUSSION THREAD

**Discussion by hughbris** [getgrav/grav#3563] [Help] [answered]
I'm out of the loop, obviously. I thought Discussions seemed pretty quiet lately when I looked the other day. I only just noticed Discourse is pretty active. What's the deal? Where should I ask/answer support questions?

**Comment by pamtbaau** [comment, marked as answer]
@hughbris, It is more likely the community has never moved to Discussions...

However Discourse is on a continuous decline unfortunately... 

**New contributors/month**: (A new contributor is someone who created their first post or like)
![Untitled](https://user-images.githubusercontent.com/5120011/158970563-84b24903-a71c-40ba-9843-97d40a1ec8b5.png)

**Number of new topics/month**:
![Untitled](https://user-images.githubusercontent.com/5120011/158972317-d4faf35a-9346-4b2b-80cd-171c9f2fbe30.png)

Considering the response to my two discussions, I guess it's due to a lack of interest in Discourse. I'm on the brink of throwing the towel.

**Reply by OleVik** [reply]
Is said towel on Discourse, the ecosystem of communities around Grav, or Grav itself? My own declining interest in Discourse stems from the forum 2.0 organization of the system itself, where I find StackOverflow-style of post-listing inherently messy and not conducive to effective tracking of discussions. I have some of the same issues with a closed, proprietary chat-system like Discord. GitHub Discussions is easy to conceptualize as an addendum to issue-tracking, but suffer greatly from being a competitor to those.

If anything, I'd perhaps want to throw my towel into something broader than a single-system, single-purpose community, as my development these days cross lines and boundaries and can't be served by just flat-file, just static, just framework, or just platform as practices or points of entry. That may not be conducive for community-building, however.

**Comment by Jogai** [comment]
Discord for software projects is imo a bad idea. There is no publicly searchable history, discussions move past your point of interest without a resolution. Its a mess if you want to stay in touch with the general conversation after leaving for a bit. GH discussions dont have that problem but then you need moderation and you dont have a means for real-time chat but moderated the right way chat is often not interesting to the larger public.

**Comment by pamtbaau** [comment]
@Jogai, Could it be you are confusing [Discourse](https://discourse.getgrav.org/) with Discord? All disadvantages you mention hold for Discord, not Discourse

**Comment by Jogai** [comment]
Whoops, yes I was. I guess what I'm saying is to keep something besides discord.

**Comment by pamtbaau** [comment]
I have no statistics on Discord, but considering the statistics shown above about Discourse and seeing people who were once active on Discourse and now on Discord, I'm afraid Discord is cannibalising Discourse. No matter its drawbacks, chat seems to be attractive to many.

**Reply by OleVik** [reply]
Chat is attractive, as is the community -and network-features Discord offers, as mentioned previously. The stats show a decline in Discourse's new, internal traffic, but not really external traffic, and certainly not quality. Which is to say that new people are likely better exposed to many more answers than previously via Google - the main driver of nearly all traffic - and it's not unexpected that new users won't have as many new questions to ask. That is, of course, without any qualified analysis of actual data, Grav's popularity in itself, whether its development has slowed, or Grav's internals become too complex.

Further, also without analysis but from daily surveillance, I find that topics and answers have increased in quality on Discourse. Many more questions are asked with thought given to formulation and more necessary information is available, and both @pamtbaau and @Karmalakas do an especially good job helping out the inquirer with getting the necessary information into the light, as well as provide pedagogical and helpful answers.

It was [the case in 2020](https://github.com/getgrav/grav/discussions/3104#discussioncomment-236660) and it is the case in 2022: Discourse - the forum - wins out in quality, longevity, and support. What is also clear is that no other direction was proposed or settled on [in 2021](https://github.com/getgrav/grav/discussions/3104#discussioncomment-238510), and GitHub Discussions never became a thing for Grav. Then again, Discourse and Discord doesn't really run on the support of the Core Team, it's almost entirely built around the community's efforts.

**Comment by Jogai** [comment]
Something that seems more attractive doesn't need to be necessarily better. Chat by nature is faster, that makes it tempting to use in the first place, but on the other hand its also more fleeting. I didn't expand on this earlier, but another big drawback of that approach is that you don't know the quality of the answer. In a forum a topic can be discussed over a longer time and that should result in a more refined answer. (In harsher words, how do you know if the random chat user that answers you at that time of the day actually has any expertise about it).

I think forums and chat should be moderated, so their purpose is not mingled. Ideally moderators should redirect users with substantive questions to the forums and forum moderators should redirect the more fleeting content to the chat.

**Reply by hughbris** [reply]
Chat is attractive to me in some situations, but our Discord hasn't been for a number of reasons.

Moderation would go a long way towards addressing one of them. There is just no fairness or accountability. The other is time zones, I'm just not in the best one for real-time help in this community. Searchability is a huge loss for the community, too.

Discord is benefitting from the network effect at this point, so it's popular beyond what users' preferences might be. Just like we all know people that want to get off mainstream social media but feel like they can't. Last I checked, Discourse isn't really getting much support from the Grav team. I say that cautiously, would be delighted if I'm wrong. The welcome sticky message is from 2017.

**Reply by OleVik** [reply]
Each platform has moderators, but the organization and moderation hasn't grown with the community or ecosystem. Discourse never got much support or attention, though the same can be said of Discord, from the Core Team. The overall lack of a strategy and purpose for the community-channels is the broader issue though. Eg., even with efforts to direct big questions to Discourse or technical issues to GitHub Issues, they tend to get solved directly in chat. On the other side, the Core Team is small and over the years have relied on the community-efforts to keep the community-channels alive. I wouldn't expect a coherent strategy to form in the next 5 years either, but I'd argue it shouldn't: A desired or necessary renewal needs to come from the bottom, not the top, as communities thrive and develop from its members.
